src/main/webapp/manage-availability.js

```diff
@@ -12,13 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** A function that adds event listeners to a DOM objects. */
+function addEventListeners() {
+    document.getElementById("timeslot-form").addEventListener('submit', event => {
+        event.preventDefault();
```

Comment:
What does this preventDefault do?

Reply:
It prevents the form from submitting and reloading the page.

Reply:
ohhh this is helpful